src/main/java/sirius/biz/web/RssFeedHelper.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.web;
+
+import com.rometools.rome.feed.synd.SyndFeed;
+import com.rometools.rome.io.FeedException;
+import com.rometools.rome.io.SyndFeedInput;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.xml.Outcall;
+import sirius.web.security.HelperFactory;
+import sirius.web.security.ScopeInfo;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.io.StringReader;
+import java.net.URL;
+
+/**
+ * Helps to obtain an {@link SyndFeed} object
+ */
+public class RssFeedHelper {
+
+    private static final String RSS_FEED_HELPER = "RSSFeedHelper";
+
+    /**
+     * Generates a helper instance per shop.
```

Comment:
remove hint to shop as it has nothing to do with this helper class?

Reply:
done, thanks.